src/shared/internal-calendar/animated-internal-calendar.style.tsx

```diff
@@ -6,7 +6,7 @@ export const AnimatedDiv = styled(animated.div)`
     position: absolute;
     top: calc(100% + 0.5rem);
     left: 0;
-    width: 100%;
+    width: calc(100% + 2px);
```

Comment:
still need to offset to the left by 1px. because `left: 0` means it still starts within the parent boundary